provision/acc_provision/acc_provision.py

```diff
@@ -172,6 +172,7 @@ def config_default():
             "interface_mtu": None,
             "second_kubeapi_portgroup": False,
             "disable_wait_for_network": False,
+            "duration_wait_for_network": "210",
```

Comment:
I don't think needs to be a string? (It's also not consistent with your example later in the patch).